src/main/webapp/WEB-INF/tags/instructor/course/studentTableModal.tag

```diff
@@ -4,10 +4,10 @@
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
       <div class="modal-header">
-        <span class="pull-left help-block">
-          Tips: After Selecting the table, <kbd>Ctrl + C</kbd> to COPY and <kbd>Ctrl + V</kbd> to PASTE to your Excel Workbook.
+        <span class="help-block">
+          Tips: After Selecting the table, <kbd>Ctrl + C</kbd> or <kbd>&#8984; + C</kbd> to COPY and
+          <kbd>Ctrl + V</kbd> or <kbd>&#8984; + V</kbd> to PASTE to your Excel Workbook.
         </span>
-
```

Comment:
This is an unrelated change.

Reply:
Should I revert this back to the original? Sorry this might have been a habit that resurfaced itself based on the style guide we used at a place I worked.

Reply:
Yes, unless you decide to combine the two tag files into one, in which case the two originals can be deleted.